Description: Fix run_tests.py with Django 1.7
 In Django 1.7, we need to call django.setup(). This patch does it to fix the
 unit tests.
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/765160
Forwarded: no
Last-Update: 2014-10-17

--- python-jingo-0.7.orig/run_tests.py
+++ python-jingo-0.7/run_tests.py
@@ -9,5 +9,9 @@ os.environ['DJANGO_SETTINGS_MODULE'] = '
 os.environ['PYTHONPATH'] = os.pathsep.join([ROOT,
                                             os.path.join(ROOT, 'examples')])
 
+import django
+if hasattr(django, 'setup'):
+    django.setup()
+
 if __name__ == '__main__':
     nose.main()
